### ISSUE TITLE
Implement HTTP Server API wrapper

### ISSUE THREAD

**Issue by fxgallego** [dolphinsmalltalk/Dolphin#492]
Add new classes wrapping the HttpApiLibrary in order to implement a new HttpServer for Dolphin suitable to run web frameworks like Seaside and others. This new server can be used as direct replacement of the Swazoo web server.

**Comment by fxgallego** [comment]
@blairmcg the first version is in this branch https://github.com/fxgallego/Dolphin/tree/DolphinHttpServer
It's is based on your last commit about UnicodeString equality and I've tested that the package works ok in a fresh DPRO image. You only need to load the package Core\Contributions\fxgallego\Dolphin HTTP Server Demo.pax.
Then evaluate "HttpServerDemo start" and open your browser at http://localhost:8080

I'd like to merge this with Dolphin base so if you agree I can move it to other folder.
I'm waiting your comments about style, names, missing functionality. I'll be adding tests on this packages.

This implementation was tested using the Seaside functional tests suite and I've implemented it's corresponding adapter, not present in this commit.
By default the server works multithreaded and to protect the image to crash by exceed on Process instances it implements a ThreadPool.
I've tested it using ab (apache benchmark) with 100,000 requests and 12 threads without problems. In my i5 laptop the server reach up to 2800 req/sec while the memory does not grow.
